docs/CHANGELOG.md

```diff
@@ -1,9 +1,30 @@
 Antares Changelog
 =================
 
+v8.3-rc1 (06/2022)
+--------------------
+### New features
+- Adequacy patch - share the unsupplied energy according to the "local matching rule". This feature was contributed by RTE-i with support from RTE, ELIA and APG #657
+- Add output variable "profit by cluster". This variable represents the difference between proportional costs and marginal costs in the area. It provides a partial answer to the question "what is the economic profit associated to a thermal cluster ?", excluding non-proportional (€/h) and startup costs (€/startup). #686
+- Splitted MPS files for Antares Xpansion, to reduce the RAM & disk footprint #643
```

Comment:
**split** instead of **splitted**

Reply:
splitted seems [to be correct](https://www.linguee.fr/anglais-francais/traduction/to+be+splitted.html)